dashboard/src/components/EventsListItem.js

```diff
@@ -55,19 +63,48 @@ class EventListItem extends React.Component {
     // eslint-disable-next-line react/forbid-prop-types
     classes: PropTypes.object.isRequired,
     Chevron: PropTypes.func.isRequired,
+    Disclosure: PropTypes.func.isRequired,
   };
 
   static defaultProps = {
     Chevron: chevronIcon,
+    Disclosure: disclosureIcon,
```

Comment:
I don't think we need to have the icons passed in as props, unlikely we'll want to change them.